web/src/PreviewPane.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import PreviewPane from "./PreviewPane"
+import renderer from "react-test-renderer"
+
+it("renders without crashing", () => {
+  let div = document.createElement("div")
+  Element.prototype.scrollIntoView = jest.fn()
+  ReactDOM.render(<PreviewPane endpoint="http://localhost:9000" />, div)
```

Comment:
@yuindustries and I talked and we should add a test here for when no endpoint is passed. The preview component should probably handle that gracefully.